src/shared/verbs.c

```diff
@@ -143,7 +143,18 @@ int dispatch_verb(int argc, char *argv[], const Verb verbs[], void *userdata) {
                         return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Unknown command verb '%s'.", name);
                 }
 
-                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Command verb required.");
+                _cleanup_free_ char *verb_list = NULL;
+
+                for (size_t i = 0; verbs[i].dispatch; i++)
+                        if (!strextend_with_separator(&verb_list, ", ", verbs[i].verb)) {
+                                log_oom_warning();
```

Comment:
i'd really just do `return log_oom()` here.

i mean, there's so little value in trying to go on here in case of oom (which btw, is a case that's impossibly hard to trigger here even), let#s just do it simple and fail here.